packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1304,6 +1304,77 @@ export const UnknownFilesystemError = {
 	hint: 'It can be caused by many things, eg. missing permissions or a file not existing we attempt to read. Check the error cause for more details.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Cannot extract the font type from the given URL.
+ */
+export const CannotExtractFontType = {
+	name: 'CannotExtractFontType',
+	title: 'Cannot extract the font type from the given URL.',
+	hint: 'Open an issue at https://github.com/withastro/astro/issues',
```

Comment:
I noticed where this error is used, and I believe we can be more specific here. For example, we can pass the extension and print it if it exists. That will help us with the triaging.